src/ast/irbuilderbpf.cpp

```diff
@@ -253,8 +253,7 @@ llvm::Type *IRBuilderBPF::GetType(const SizedType &stype)
         ty = getInt8Ty();
         break;
       default:
-        LOG(FATAL) << stype.GetSize()
-                   << " is not a valid type size for GetType";
+        LOG(BUG) << stype.GetSize() << " is not a valid type size for GetType";
```

Comment:
These errors all seemed like internal bugs. Please correct me if I'm wrong.

Reply:
I believe that you're right.